Implemented support for lower threshold

roberthorlings opened the pull request:
For BMI measurements, I want to show an upper and lower boundary. This seemed not to be supported, so I took the liberty of adding support myself. Hopefully you can use it as well.

roberthorlings committed:
Implemented support for lower threshold

jasperSpeicher commented:
Thank you @roberthorlings
Do you also need to show the line for the minimum?
Jasper

roberthorlings commented:
The line is indeed shown in the graph, which is done in line 555.

roberthorlings committed:
Added support for legends for line charts as well

roberthorlings committed:
Possibility to configure a function to change the tooltip text.

Also, the number of decimal places can be specified in configuration. This
is useful when no custom function is specified. The number of decimal places
defaults to 1.

roberthorlings committed:
Merge branch 'features/content-formatter'

roberthorlings committed:
Merge branch 'features/legends'

roberthorlings committed:
Added combined and minified scripts

roberthorlings committed:
Enabled separate tooltips for otherwise grouped measurements

roberthorlings committed:
Merge branch 'features/separate-tooltips'

roberthorlings committed:
Creating combined scripts

roberthorlings committed:
Bugfix in separating tooltips from groups

roberthorlings committed:
Merge branch 'features/separate-tooltips'

roberthorlings committed:
Added support for multiple thresholds within a measurement type

roberthorlings committed:
Made the visibility of thresholds configurable

roberthorlings committed:
Merge branch 'features/improved-threshold-support'

roberthorlings committed:
Creating combined scripts

roberthorlings committed:
Added datapoint id to tooltip
